src/redux/actions/timetable/lectureFocus.ts

```diff
@@ -12,13 +12,19 @@ export const SET_REVIEWS = `${BASE_STRING}SET_REVIEWS` as const;
 export const SET_MULTIPLE_FOCUS = `${BASE_STRING}SET_MULTIPLE_FOCUS` as const;
 export const CLEAR_MULTIPLE_FOCUS = `${BASE_STRING}CLEAR_MULTIPLE_FOCUS` as const;
 
+type LectureFocusFromListOrTable = LectureFocusFrom.LIST | LectureFocusFrom.TABLE;
+
 export function reset() {
   return {
     type: RESET,
   };
 }
 
-export function setLectureFocus(lecture: Lecture, from: LectureFocusFrom, clicked: boolean) {
+export function setLectureFocus(
+  lecture: Lecture,
+  from: LectureFocusFromListOrTable,
```

Comment:
(질문) `LectureFocusFrom.MULTIPLE` 케이스는 `setMultipleFocus`에서 처리되는 건가요?

Reply:
따로 타입 정의하지 말고 바로 쓰는 게 더 직관적일 수도 있다는 생각
```suggestion
  from: LectureFocusFrom.LIST | LectureFocusFrom.TABLE,
```